Userland/Libraries/LibTest/Macros.h

```diff
@@ -46,6 +46,14 @@ u64 randomized_runs();
         }                                                                                                     \
     } while (false)
 
+// Performs both run-time EXPECT_EQ and compile-time static_assert.
+// This also ensures that both a and b are constant expressions (e.g. usable in constexpr/consteval contexts).
+#define EXPECT_EQ_AND_STATIC_ASSERT(a, b) \
+    do {                                  \
+        static_assert(a == b);            \
+        EXPECT_EQ(a, b);                  \
```

Comment:
pretty sure this get's fully constant folded out of existence...

So not sure how valuable these helpers are